packages/terra-toolkit/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "terra-toolkit",
+  "version": "0.1.0",
+  "description": "Utilities to help when developing terra modules",
+  "main": "lib/index.js",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-toolkit.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-toolkit",
+    "UI"
+  ],
+  "scripts": {
+    "compile": "../../node_modules/.bin/babel src --out-dir lib --copy-files",
+    "lint": "npm run lint:js",
+    "lint:js": "../../node_modules/.bin/eslint **/*.js, **/*.jsx",
+    "pretest": "npm run lint",
+    "release:major": "npm test && npm run compile && npm version major -m \"Released version %s\" && npm publish && git push",
+    "release:minor": "npm test && npm run compile && npm version minor -m \"Released version %s\" && npm publish && git push",
+    "release:patch": "npm test && npm run compile && npm version patch -m \"Released version %s\" && npm publish && git push",
+    "test": "../../node_modules/.bin/nightwatch -e default-tiny,default-small,default-medium,default-large,default-huge,default-enormous"
+  },
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-toolkit/issues"
+  },
+  "dependencies": {
+    "nightwatch": "^0.9.12",
+    "phantomjs-prebuilt": "^2.1.14",
+    "selenium-server-standalone-jar": "3.1.0",
+    "terra-application": "^0.2.0",
```

Comment:
For the tests I'm running in this project I'm still including those packages to simulate how the tests will be run in other packages.  We can discuss whether that's necessary or not.